.gitignore

```diff
@@ -1 +1,2 @@
 node_modules
+dist
```

Comment:
You are, of course, technically right... but... grumble grumble, the distinction will probably never be noticed in this project, grumble